.github/workflows/podman-test.yaml

```diff
@@ -0,0 +1,26 @@
+name: GitHub Actions Demo
+
+on:
+  pull_request:
+    branches:    
+      - main
+
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
```

Comment:
To avoid duplication, I think the `setup-go` Action is able to pick the Go version specified in `go.mod`. See https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file